python-threatexchange/README.md

```diff
@@ -94,26 +94,73 @@ The `threatexchange` cli is designed to rapidly demonstrate the value of the lib
 
 ## Usage
 
-While the CLI was designed for use with signal exchanges, it also comes with a built-in copy of data that is loaded 
+While the CLI was designed for use with signal exchanges. The normal flow is roughly: 
+1. configure collaborations
+2. fetch from APIs
+3. build indices 
+4. match data
+5. contribute labels and data
 
 ```bash
 $ threatexchange --help  # The help should give a decent overview of functionalities
 
-# You can immediately begin matching against text data
+# Step 1: We can skip this step if using data
```

Comment:
unsure: is this missing a word? e.g. "sample data", "local data", "testing data"

Reply:
the sample data

I accidentally a word